### ISSUE TITLE
No helmet.contentSecurityPolicy.getDefaultDirectives() method

### ISSUE THREAD

**Issue by thomasdingemanse** [fastify/fastify-helmet#108]
## 🐛 Bug Report

The [helmet documentation](https://helmetjs.github.io/), section **reference**, under **helmet.contentSecurityPolicy(options)**, gives the following information:

> If no directives are supplied, the following policy is set (whitespace added for readability):
> 
> ```
> default-src 'self';
> base-uri 'self';
> block-all-mixed-content;
> font-src 'self' https: data:;
> frame-ancestors 'self';
> img-src 'self' data:;
> object-src 'none';
> script-src 'self';
> script-src-attr 'none';
> style-src 'self' https: 'unsafe-inline';
> upgrade-insecure-requests
> ```
> 
> You can fetch this default with `helmet.contentSecurityPolicy.getDefaultDirectives()`.

I've tried doing this with fastify-helmet, but I get the following error:
```
    ...fastifyHelmet.contentSecurityPolicy.getDefaultDirectives(),
                                           ^
TypeError: Cannot read property 'getDefaultDirectives' of undefined
```

## To Reproduce

Create a fastify server and register the fastify-helmet plugin. Try to reference the `getDefaultDirectives()` method in the options object. Start the server to see the error message.

```js
const fastify = require('fastify')({ logger: true })
const helmet= require('fastify-helmet')

fastify.get('/', function (request, reply) {
  reply.send({ hello: 'world' })
})

fastify.register(helmet, {
  contentSecurityPolicy: {
    directives: {
      ...helmet.contentSecurityPolicy.getDefaultDirectives()
    }
  }
})

fastify.listen(3000, function (err, address) {
  if (err) {
    fastify.log.error(err)
    process.exit(1)
  }
  fastify.log.info(`server listening on ${address}`)
})
```

A workaround for this issue is to specify all of the CSP defaults manually, in addition to any custom modifications. However, it would be a much more elegant solution if this worked as described in the helmet documentation. I think adding a `contentSecurityPolicy` object with a `getDefaultDirectives()` method that just calls the original `helmet.contentSecurityPolicy.getDefaultDirectives()` and returns the result should solve this, but I'm not sure how additional exports can be defined for fastify plugins.

## Expected behavior

I expected the server to start normally without throwing any errors. Fastify-helmet should have a `contentSecurityPolicy.getDefaultDirectives()` method that behaves the same as it does in helmet.

## Your Environment

- *node version*: 14.15.1
- *fastify version*: 3.9.2
- *fastify-helmet version*: 5.1.0
- *os*: Windows

**Comment by mcollina** [comment]
Would you like to send a Pull Request to address this issue? Remember to add unit tests.

**Comment by thomasdingemanse** [comment]
Yes, I'll take a shot at it!

**Comment by climba03003** [comment]
This issue should be closed as #109 is accepted.
